gordon_gcp/plugins/enricher.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2017 Spotify AB
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Client module to enrich an event message with any missing information (
+such as IP addresses to a new hostname) and to generate the desired
+record(s) (e.g. ``A`` or ``CNAME`` records). Once an event message is
+done (either successfully enriched, or encountered errors along the
```

Comment:
This is a bit of a nitpick, but I don't think part of the parenthesis is in agreement with the "is" in the sentence: "Once an even message _is_ done ... _or encountered errors along the way_", which makes it less clear.

My proposal:
Once an event message is done (either successfully enriched or met with errors along the way)...

This also pops up in the Publisher's module docstring.

Reply:
I really like + appreciate these nitpicks. Getting docs right is important!